processor/intervalprocessor/config.go

```diff
@@ -4,21 +4,31 @@
 package intervalprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/intervalprocessor"
 
 import (
+	"errors"
 	"time"
 
 	"go.opentelemetry.io/collector/component"
 )
 
+var (
+	ErrInvalidIntervalValue     = errors.New("invalid interval value")
+	ErrInvalidMaxStatenessValue = errors.New("invalid max_stateless value")
```

Comment:
Ah oops. This should be deleted as well. I removed the staleness config, because we clear the state every export. So IMO, it's excessive to also have staleness.

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32054/commits/c8f1d970d930e09a5f18a9ee16012488fc579c84